src/AeSdkAepp.ts

```diff
@@ -92,11 +92,19 @@ export default class AeSdkAepp extends AeSdkBase {
    */
   async connectToWallet(
     connection: BrowserConnection,
-    { connectNode = false, name = 'wallet-node', select = false }:
-    { connectNode?: boolean; name?: string; select?: boolean } = {},
-  ): Promise<WalletInfo> {
+    { connectNode = false, name = 'wallet-node' }: { connectNode?: boolean; name?: string } = {},
+  ): Promise<WalletInfo & { node?: NodeRpc }> {
     if (this.rpcClient != null) throw new AlreadyConnectedError('You are already connected to wallet');
     let disconnectParams: any;
+
+    const updateNetwork = (params: Network): void => {
+      if (connectNode) {
+        if (params.node == null) throw new RpcConnectionError('Missing URLs of the Node');
+        this.addNode(name, new Node(params.node.url), true);
+      }
+      this.onNetworkChange(params);
```

Comment:
Minor refactoring
```suggestion
  if (!connectNode) {
    return;
  }

  if (!params.node?.url) {
    throw new RpcConnectionError('Missing URLs of the Node');
  }

  this.addNode(name, new Node(params.node.url), true);
  this.onNetworkChange(params);
```

Reply:
`onNetworkChange` needs to be called regardless `connectNode`, and it can't be moved above because I expect it to be fired after network switching complete. Casting to boolean using `!` is not allowed by linter. I'm accepting `node?.url` part.